docs/getting-started/linux.rst

```diff
@@ -81,4 +78,4 @@ The following steps will install `dnvm <https://github.com/aspnet/home#running-a
 If you have trouble installing dnvm, consult this `Getting Started guide <http://dotnet.github.io/core/getting-started/>`_.
 
 
-.. include:: xplat.rst
\ No newline at end of file
+.. include:: x-plat/guide.rst
```

Comment:
It doesn't. I just moved into x-plat so the files in this folder represent articles and includes come from the x-plat subfolder. I have noticed, btw, that the TOC on the left includes a :heavy_plus_sign:  button next the article title, even though there no subitems. Not sure how to fix this...